### DISCUSSION TITLE
Potential migration to GitHub Discussions

### DISCUSSION THREAD

**Discussion by rmcdermo** [firemodels/fds#11520] [Polls]
We are considering a move of the discussion forum from Google Groups to GitHub Discussions to coincide roughly with release of FDS 7.  The migration would not be abrupt; we would maintain both forums for a reasonable period of time.

GitHub Discussions offers several advantages:
- It would consolidate information management for the FDS project, which would be helpful to the developers and ultimately mean we could spend more time on important development tasks.
- It allows for seamless conversion between Discussions and Issues.  (Many new users sometimes confuse Issues and Discussions and send queries to the Issues page anyway.)
- GitHub Discussions is under active development and well supported by GitHub, enabling features like this poll, for example.

**Comment by marcosvanella** [comment]
Is there a way to pin it to the discussions landing page? Or make an announcement on the poll?

**Comment by mcgratta** [comment]
GoogleGroups dropped the feature of pinning the thread at the top of the list.

**Reply by rmcdermo** [reply]
An example of them reducing support for Groups... someday it will go away like Google Code.

**Comment by akilan-agil** [comment]
Moving the discussion forum from Google Groups to GitHub Discussions for FDS 7 release is a good plan that offers several advantages, including improved information management and seamless conversion between Discussions and Issues. Maintaining both forums during the transition is a wise move.
